src/apps/intermediate-list/stackable-fees/fee-status.tsx

```diff
@@ -15,7 +14,7 @@ const FeeStatus: FC<FeeStatusProps> = ({ dueDate, reasonMessage }) => {
   return (
     <div>
       <div className="list-reservation__deadline">
-        <StatusBadge dueDate={dueDate} dangerText={reasonMessage} />
+        <div className="status-label status-label--danger">{reasonMessage}</div>
```

Comment:
This seems like a generic way to show a "danger message" and it is used multiple places?
In that case I think it should be a seperate component we can reuse.

Reply:
I can change it back to using the generic component, if you like, it just doesnt ever change color (the fee-status), which is sort of what StatusBadge is good at